client/src/components/Styles.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+import Style from './Style.jsx';
+
+const Styles = styled.div`
```

Comment:
I am starting to understand styled components a little bit. It seems you write the css directly in the JSX file for a specific component? I can see why that's convenient!